apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,17 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
+	// +optional
+	Enable bool `json:"enable,omitempty"`
+
+	// +optional
+	LocalQueueSelector *metav1.LabelSelector `json:"localQueueSelector,omitempty"`
```

Comment:
Can you please add a comment about what this field does?

These are generated as reference docs so it is important to treat this as a user facing API.